docs/cli/kops.md

```diff
@@ -0,0 +1,42 @@
+## kops
+
+kops is kubernetes ops
+
+### Synopsis
+
+
+kops is kubernetes ops.
```

Comment:
Looking at other packages (like Kubernetes), seems like you would have the sentence from the `Short` field as the first sentence in the `Long` field, with additional sentences. I think we should keep it as it is, and maybe try to add more info to the `Long` field where possible.